docs/csharp/language-reference/operators/greater-than-operator.md

```diff
@@ -11,17 +10,25 @@ helpviewer_keywords:
 ms.assetid: 26d3cb69-9c0b-4cc5-858b-5be1abd6659d
 ---
 # &gt; Operator (C# Reference)
-All numeric and enumeration types define a "greater than" relational operator (`>`) that returns `true` if the first operand is greater than the second, `false` otherwise.  
-  
-## Remarks  
- User-defined types can overload the `>` operator (see [operator](../../../csharp/language-reference/keywords/operator.md)). If `>` is overloaded, [<](../../../csharp/language-reference/operators/less-than-operator.md) must also be overloaded.
-  
-## Example  
- [!code-csharp[csRefOperators#29](../../../csharp/language-reference/operators/codesnippet/CSharp/greater-than-operator_1.cs)]  
-  
-## See Also
-
-- [C# Reference](../../../csharp/language-reference/index.md)  
-- [C# Programming Guide](../../../csharp/programming-guide/index.md)  
-- [C# Operators](../../../csharp/language-reference/operators/index.md)  
-- [explicit](../../../csharp/language-reference/keywords/explicit.md)
+
+The "greater than" relational operator `>` returns `true` if its first operand is greater than its second operand, `false` otherwise. All numeric and enumeration types support the `>` operator. For operands of the same [enum](../keywords/enum.md) type, the corresponding values of the underlying integral type are compared.
+
+The following example demonstrates the usage of the `>` operator:
+
+[!code-csharp-interactive[greater than example](~/samples/snippets/csharp/language-reference/operators/GreaterAndLessOperatorsExamples.cs#Greater)]
+
```

Comment:
This comment applies to all articles.  Following the example, I'd add the following note and link:

> Any relational comparison where one of the values is `NaN` (not a number) returns `false.`  The `NaN` value is neither greater than, less than, nor equal to any other `double` value.  For more information and examples, see the <xref:System.Double.NaN> reference article.

Reply:
@BillWagner thank you for sharing the developer feedback. I've added the note; it's slightly different from your suggestion. Please check the latest commit and let me know if the text should be updated.

Also, I've added examples of comparison with the `NaN` value: dotnet/samples#525

Reply:
That's perfect @pkulikov 

I'll approve this and :shipit: now.